.streamlit/config.toml

```diff
@@ -0,0 +1,195 @@
+# Default: 200
+maxUploadSize = 200
+
+# Enables support for websocket compression.
+# Default: true
+enableWebsocketCompression = true
+
+[browser]
+
+# Internet address where users should point their browsers in order to connect to the app. Can be IP address or DNS name and path.
+# This is used to: - Set the correct URL for CORS and XSRF protection purposes. - Show the URL on the terminal - Open the browser - Tell the browser where to connect to the server when in liveSave mode.
+# Default: 'localhost'
+serverAddress = "localhost"
+
+# Whether to send usage statistics to Streamlit.
+# Default: true
+gatherUsageStats = true
+
+# Port where users should point their browsers in order to connect to the app.
+# This is used to: - Set the correct URL for CORS and XSRF protection purposes. - Show the URL on the terminal - Open the browser - Tell the browser where to connect to the server when in liveSave mode.
+# Default: whatever value is set in server.port.
+serverPort = 8501
+
+[mapbox]
+
+# Configure Streamlit to use a custom Mapbox token for elements like st.pydeck_chart and st.map. To get a token for yourself, create an account at https://mapbox.com. It's free (for moderate usage levels)!
+# Default: ""
+token = ""
+
+[deprecation]
+
+# Set to false to disable the deprecation warning for the file uploader encoding.
+# Default: "True"
+showfileUploaderEncoding = "True"
+
+# Set to false to disable the deprecation warning for the image format parameter.
+# Default: "True"
+#
+# DEPRECATED.
+# The format parameter for st.image has been removed.
+# This option will be removed on or after 2021-03-24.
+#
+# showImageFormat = "True"
+
+# Set to false to disable the deprecation warning for using the global pyplot instance.
+# Default: "True"
+showPyplotGlobalUse = "True"
+
+[s3]
+
+# Name of the AWS S3 bucket to save apps.
+# Default: (unset)
+#bucket =
+
+# URL root for external view of Streamlit apps.
+# Default: (unset)
+#url =
+
+# Access key to write to the S3 bucket.
+# Leave unset if you want to use an AWS profile.
+# Default: (unset)
+#accessKeyId =
+
+# Secret access key to write to the S3 bucket.
+# Leave unset if you want to use an AWS profile.
+# Default: (unset)
+#secretAccessKey =
+
+# The "subdirectory" within the S3 bucket where to save apps.
+# S3 calls paths "keys" which is why the keyPrefix is like a subdirectory. Use "" to mean the root directory.
+# Default: ""
+keyPrefix = ""
+
+# AWS region where the bucket is located, e.g. "us-west-2".
+# Default: (unset)
+#region =
+
+# AWS credentials profile to use.
+# Leave unset to use your default profile.
+# Default: (unset)
+#profile =
+
+
+[theme]
+primaryColor="#ee133f"
+backgroundColor="#FFFFFF"
+secondaryBackgroundColor="#cfe2f1"
+textColor="#262730"
+font="sans serif"
+
+
+# Font family for all text in the app, except code blocks. One of "sans serif", "serif", or "monospace".
+# Default: "sans serif"
+font="calibri"
```

Comment:
I had to remove this line in order to use this `config.toml` file, otherwise was getting an error message that **font** was duplicated in line 189.